content/documentation/pages/1-installation/1-local/2-docker-customize.md

```diff
@@ -10,91 +10,175 @@ The Docker Compose [installation](%currentPath%/installation/local/docker) guide
+The [docker-compose-postgres.yml](https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%github-tag%/spring-cloud-dataflow-server/docker-compose-postgres.yml) configures using `PostgreSQL` instead of `MySQL` for both Spring Cloud Data Flow and SKipper. It disables the default `mysql` service, adds a new `postgres` service and overrides the Data Flow and Skipper configurations to use the postgres:
+
+<!--TABS-->
+
+<!--Linux / OSX-->
 
 ```bash
 wget https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%github-tag%/spring-cloud-dataflow-server/docker-compose-postgres.yml
 docker-compose -f ./docker-compose.yml -f ./docker-compose-postgres.yml up
 ```
 
-### RabbitMQ Instead of Kafka
-
-[docker-compose-rabbitmq.yml](https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%github-tag%/spring-cloud-dataflow-server/docker-compose-rabbitmq.yml) - Use RabbitMQ instead of Kafka as message broker. It disables the default `kafka` and `zookeeper` services, add a new `rabbitmq` service and override the `dataflow-server`'s service binder configuration to RabbitMQ (e.g. `spring.cloud.dataflow.applicationProperties.stream.spring.rabbitmq.host=rabbitmq`). Finally overrides the `app-import` service to register the rabbit apps:
+<!--Windows-->
 
 ```bash
-wget https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%github-tag%/spring-cloud-dataflow-server/docker-compose-rabbitmq.yml
-docker-compose -f ./docker-compose.yml -f ./docker-compose-rabbitmq.yml up
+curl https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%github-tag%/spring-cloud-dataflow-server/docker-compose-postgres.yml -o docker-compose-postgres.yml
+docker-compose -f .\docker-compose.yml -f .\docker-compose-postgres.yml up
 ```
 
-### Multi-platform support
+<!--END_TABS-->
+
+The `docker-compose-postgres.yml` configurations expose the following container ports to the host machine:
+
+| Host ports | Container ports | Description                                                                                                           |
+| ---------- | --------------- | --------------------------------------------------------------------------------------------------------------------- |
+| 5432       | 5432            | PostgreSql DB server port. Use jdbc:postgresql://localhost:5432/dataflow to connect to the DB form your local machine |
+
+## RabbitMQ Instead of Kafka
+
+The [docker-compose-rabbitmq.yml](https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%github-tag%/spring-cloud-dataflow-server/docker-compose-rabbitmq.yml) configures using RabbitMQ instead of Kafka as message broker. It disables the default `kafka` and `zookeeper` services, add a new `rabbitmq` service and override the `dataflow-server`'s service binder configuration to RabbitMQ (e.g. `spring.cloud.dataflow.applicationProperties.stream.spring.rabbitmq.host=rabbitmq`). Finally overrides the `app-import` service to register the rabbit apps:
```

Comment:
configures using -> configures

Reply:
add->adds,  override->overrides
Finally, it overrides

Reply:
:)